src/java.base/share/classes/java/net/doc-files/net-properties.html

```diff
@@ -265,15 +266,22 @@ <H2>Unix domain sockets</H2>
 	(eg Windows) there is no default value. Either way, it is always possible
 	to override the networking property with a system property of the same name
 	set on the command line. If neither of the networking nor system property
-	are set, then some systems may check a commonly used environment variable as temporary
-	directory.
+	are set, then some systems (eg Windows)  may check a commonly used environment 
+	variable as temporary directory.
 	<LI><P><B>{@systemProperty java.io.tmpdir}</b> If the previous step fails to locate
```

Comment:
nit again (two places): `<LI><P><B>` should probably be `<li><p><b>` especially since the `<B>` tag is closed with `</b>`